### PR TITLE
Update documentation for correctness and content

### LINKED ISSUE #2159: [Question] How are docs generated?
I would like to ask about how is the documentation of https://turfjs.org/ generated?

### PR DESCRIPTION
**Pull request by mfedderly**
Fixes #2159

Made a bunch of documentation changes to start to clean up the docs. I added some comments to individual changes inline. This isn't really my strong suit so please nitpick anything I did or did not touch in this PR and I'll be happy to keep iterating here.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,17 +89,37 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ## Publishing
 
-Install lerna:
-
-```bash
-$ npm install -g lerna@2.0.0-beta.34
-```
+### Prerelease
```
Comment (mfedderly) [inline CONTRIBUTING.md:92]: This is just replacing the old publishing stuff in CONTRIBUTING with the content from the now-deleted RELEASING

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -143,6 +160,10 @@ Building Docs: @turf/boolean-clockwise
 ....
 ```
 
+### Public website
```
Comment (mfedderly) [inline CONTRIBUTING.md:163]: This is the actual #2159 update

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,14 +16,11 @@ The high-level structure of Turf is undergoing discussion at [#1428](https://git
 
 ## Code Style
 
-To ensure code style, at the `turf` root level run
```
Comment (mfedderly) [inline CONTRIBUTING.md:21]: This section was pretty out of date, so I rewrote it with some relevant bits.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
-### :bug: [How to report a bug](http://polite.technology/reportabug.html)
+## Bug Reporting :bug:
 
-## Please note
```
Comment (mfedderly) [inline CONTRIBUTING.md:3]: Reformatted this, basically.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
+One of the most important things you can do is report bugs. Please reference [how to report a bug](http://polite.technology/reportabug.html) and follow the issue templates when adding new bugs.
 
-The high-level structure of Turf is undergoing discussion at [#1428](https://github.com/Turfjs/turf/issues/1428). Currently (June 2018), there is a partial conversion to Typescript, and the contribution documentation does not completely reflect the current status.
```
Comment (mfedderly) [inline CONTRIBUTING.md:5]: Removed this. It seems like we're relatively stable on half typescript half javascript for the time being.

File: README.md [hunk 6]
```diff
@@ -43,19 +48,9 @@ You can also include it directly from a CDN:
 <script src="https://cdn.jsdelivr.net/npm/@turf/turf@6/turf.min.js"></script>
 ```
 
-You can create light-weight turf builds with only the functions you need using the [turfjs-builder UI](https://turfjs-builder.herokuapp.com/) or using browserify as described below.
```
Comment (mfedderly) [inline README.md:46]: turfjs-builder seems to be using an old version of turf and I cannot really recommend it any more.

File: README.md [hunk 7]
```diff
@@ -43,19 +48,9 @@ You can also include it directly from a CDN:
...
-
-### Browserify
```
Comment (mfedderly) [inline README.md:48]: We previously removed some browserify references somewhere else. I don't know that many people use browserify at this point.

**Review by mfedderly — COMMENTED** [review]
As a side note, I really like the split that README is focused on a TurfJS consumer and CONTRIBUTING is focused on someone ready to file bugs or write code in the project.

File: README.md (hunk 6 above)
Comment (rowanwins) [reply README.md:46]: Yeah I think I'll archive that bad boy

**Review by rowanwins — APPROVED** [review]
Brilliant!

**Pull request merged**